server/src/routes/api/v1/parkReviewsRouter.js

```diff
@@ -34,4 +34,31 @@ parkReviewsRouter.post("/", async (req, res) => {
   }
 });
 
+parkReviewsRouter.delete("/:reviewId", async (req, res) => {
+  try {
+    const reviewId = req.params.reviewId;
+    await Review.query().deleteById(reviewId);
+    return res.status(204).json({});
```

Comment:
We should always send something back as JSON, even if it's a simple object like { success: "true" } for us developers. Here however, I would recommend after deleting the given review, we should query for it's Park and the remaining current reviews to send to the front end. It does require an additional query to the database, but our backend is much more efficient at handling that type of logic than our front end. Where possible we want to put together all the data we need on the back end so that the front end can be concerned with displaying the info. This also means you can calculate the new Review average here on the backend